app/pb_kits/playbook/pb_date_picker/_date_picker.scss

```diff
@@ -0,0 +1,76 @@
+// @import "~flatpickr/dist/flatpickr.css";
+@import "../pb_caption/caption_mixin";
+@import "../pb_textarea/textarea_mixin";
+@import "../tokens/colors";
+@import "./sass_partials/flatpickr_styles";
+
+[class^=pb_date_picker_kit] {
+  // input styles
+  [class^=pb_caption_kit] {
```

Comment:
Can we add a custom class to the caption kit so we don't have to do crazy class lookups?

Reply:
Added `.caption_selector` class name to caption kit.